### DISCUSSION TITLE
About setting up retained message on republish rule or mqtt bridge

### DISCUSSION THREAD

**Discussion by goriummaximum** [emqx/emqx#7071] [Q&A]
Hi,

spec:
EMQX 4.4.0
Docker

I have setting up the EMQX bridge to another MQTT broker, using ```emqx_bridge_mqtt.conf``` plugins and using republish rule to another topic. The message receive is OK and now I want to configure the MQTT packet to be retained when republishing, or when bridging is OK. I have not found any tutorial on the documentation. Is there any way to do that?

Many thanks.

**Comment by Machalkas** [comment]
Hi.
Same question, but for EMQX 5.2.

In docs, there is a "retain" field:
![image](https://github.com/emqx/emqx/assets/52866342/34a0a756-980f-482d-b15d-8af42e455c5c)

But I don't have one:
![image](https://github.com/emqx/emqx/assets/52866342/c9e13d2b-3379-4284-83e4-230a8b9e30ea)
